### ISSUE TITLE
False positives of 'value assigned to field <XYZ> is unused'

### ISSUE THREAD

**Issue by daurnimator** [mpeterv/luacheck#53]
See https://travis-ci.org/daurnimator/lua-http/jobs/112572318

```
Checking http/tls.lua                             2 warnings
    http/tls.lua:331:2: value assigned to field TLS_PSK_WITH_AES_128_GCM_SHA256 is unused
    http/tls.lua:332:2: value assigned to field TLS_PSK_WITH_AES_256_GCM_SHA384 is unused
```

The code in question: https://github.com/daurnimator/lua-http/blob/3fe22b16bd8351c4796e1eb0a3a6ab512dfda40d/http/tls.lua#L331

**Comment by mpeterv** [comment]
Looks like these keys are overwritten on lines 351-352?

**Comment by daurnimator** [comment]
Ah ha! sorry for the noise.
